### ISSUE TITLE
Parallelize build

### ISSUE THREAD

**Issue by cnixbtc** [breez/breez-sdk-swift#4]
Currently, running the workflow takes about an hour. We could parallelize building the binaries for the different architectures by running them in [separate jobs in parallel](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow).

This would probably improve build speed quite a lot. To do this, we would need separate make targets for each architecture instead of building them all one after each other [like we do now](https://github.com/breez/breez-sdk/blob/main/libs/sdk-bindings/makefile#L19).

See [here](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts) for how to share build artifacts between jobs.

**Comment by cnixbtc** [comment]
Closing in favor of #11 (the main repo already has this optimization).
